backend/api/views/busStop.py

```diff
@@ -69,3 +55,29 @@ def save_stop_to_db(stop_dict):
         routes=stop_dict.get("routes"),
     )
     busStop.save()
+
+
+@busStop.route("/bus-stops", methods=["DELETE"])
+def clear_stops():
+    """
+    DELETE method which wraps the delete stops collection function as
+    an API endpoint.
+    """
+    try:
+        count = delete_stop_collection()
+        return create_response(
+            status=200, message="Success! Deleted " + str(count) + " records."
+        )
+    except Exception as e:
+        return create_response(
+            status=500, message="Could not clear collection: " + repr(e)
+        )
+
+
+def delete_stop_collection():
+    """
+    Helper function to delete stop collection in db.
+    """
+    count = len(BusStop.objects())
+    [stop.delete() for stop in BusStop.objects()]

```

Comment:
What's the purpose of creating a list here? Couldn't this just be a regular for loop?